rollingpin/deploy.py

```diff
@@ -118,21 +119,50 @@ def signal_handler(sig, _):
                 yield self.event_bus.trigger("build.begin")
 
                 try:
-                    build_command = ["build"] + components
```

Comment:
Previously all components were passed through for building. Were some of those builds noops, and now will not even be sent to a build host for building?